newIDE/app/src/AssetStore/AssetStoreFilterPanel.js

```diff
@@ -0,0 +1,309 @@
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <Accordion key={filterKey} defaultExpanded>
+          <AccordionHeader>
+            <Text displayInlineAsSpan>{title}</Text>
+          </AccordionHeader>
+          <AccordionBody>
+            <ColumnStackLayout>
+              {choices.map(tag => (
+                <InlineCheckbox
+                  key={tag.value}
+                  label={tag.label ? i18n._(tag.label) : tag.value}
+                  checked={isChoiceChecked(tag.value)}
+                  onCheck={(e, checked) => setChoiceChecked(tag.value, checked)}
+                />
+              ))}
+            </ColumnStackLayout>
+          </AccordionBody>
+        </Accordion>
+      )}
+    </I18n>
+  );
+};
+
+type SetFilterProps = {|
+  filterKey: string,
+  title: ?React.Node,
+  choices: Choice[],
+  values: Set<string>,
+  setValues: (Set<string>) => void,
+|};
+
+const SetFilter = ({
+  filterKey,
+  title,
+  choices,
+  values,
+  setValues,
+}: SetFilterProps) => {
+  return (
+    <MultipleChoiceFilter
+      filterKey={filterKey}
+      title={title}
+      choices={choices}
+      isChoiceChecked={choice => values.has(choice)}
+      setChoiceChecked={(choice, checked) => {
+        if (checked) {
+          values.add(choice);
+        } else {
+          values.delete(choice);
+        }
+        setValues(values);
+      }}
+    />
+  );
+};
+
+type TagFilterProps = {|
+  filterKey: string,
+  title: ?React.Node,
+  choices: Choice[],
+  searchFilter: TagAssetStoreSearchFilter,
+  setSearchFilter: TagAssetStoreSearchFilter => void,
+  onFilterChange: () => void,
+|};
+
+const TagFilter = ({
+  filterKey,
+  title,
+  choices,
+  searchFilter,
+  setSearchFilter,
+  onFilterChange,
+}: TagFilterProps) => {
+  return (
+    <SetFilter
+      filterKey={filterKey}
+      title={title}
+      choices={choices}
+      values={searchFilter.tags}
+      setValues={values => {
+        setSearchFilter(new TagAssetStoreSearchFilter(values));
+        onFilterChange();
+      }}
+    />
+  );
+};
+
+type RangeFilterProps = {|
+  filterKey: string,
+  title: ?React.Node,
+  min: number,
+  max: number,
+  step: number,
+  scale: number => number,
+  range: [number, number],
+  setRange: ([number, number]) => void,
+|};
+
+const RangeFilter = ({
+  filterKey,
+  title,
+  min,
+  max,
+  scale,
+  step,
+  range,
+  setRange,
+}: RangeFilterProps) => {
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <Accordion key={filterKey} defaultExpanded>
+          <AccordionHeader>
+            <Text displayInlineAsSpan>{title}</Text>
+          </AccordionHeader>
+          <AccordionBody>
+            <Slider
+              value={range}
+              min={min}
+              max={max}
+              step={step}
+              scale={scale}
+              marks={true}
+              valueLabelDisplay="auto"
+              onChange={(event, newValue) => setRange(newValue)}
+            />
+          </AccordionBody>
```

Comment:
I'm not sure what is the bet way to add some margin on both sides of the slider.

Reply:
I'd suggest adding custom style to the slider, the same way we've done for other material ui components.
(search in the code `makeStyles()`)
I would add some padding left / right to the `root` https://v4.mui.com/api/slider/#css)

Reply:
I would recommend to go for a 
`<Column><Line noMargin><Slider ...></Line></Column>`

Reply:
A slider by default, like any component, takes all the width available, which is good. It's the parent job to add layouting with margins (i.e: Line, Column)